.github/workflows/release.yml

```
@@ -67,6 +67,14 @@ jobs:
             Docker: `docker pull ${{ steps.package.outputs.address }}`
           draft: false
           prerelease: false
+      - id: upload_release_package
+        name: Upload buildpackage to release
+        uses: svenstaro/upload-release-action@v2
```

From a security (since this action has access to the GitHub token) and determinism point of view, I think we should pin this third-party action to an exact patch version (we have Dependabot configured to update actions, so it will be easy to keep up to date). 

```suggestion
        uses: svenstaro/upload-release-action@v2.2.1
```